blockstack-sdk/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,37 @@
+<resources>
+
+    <!-- Refactor to Blockstack so it doesn't show on both-->
+    <style name="Theme.Blockstack" parent="Theme.MaterialComponents.Light.DarkActionBar">
```

Comment:
Didn't we ended up with a light action bar?

Reply:
We ended up with a white one yes, by style. Should I avoid using DarkActionBar ?